.circleci/config.yml

```diff
@@ -480,6 +480,13 @@ jobs:
           command: yarn run test-ios-e2e
           when: always
 
+      - restore-cache:
+          keys:
+            - v1-cocoapods-{{ checksum "template/ios/Podfile" }}
+            - v1-cocoapods-
+
+      - run: pod setup
```

Comment:
This command sets up the `~/.cocoapods` directory, and updates it if needed.

If there's a cache hit, `pod setup` should only take a few seconds to download what's new as opposed to 7+ minutes for the entire repo.